src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs

```diff
@@ -27,7 +27,6 @@ public void InitialBuild(bool fullFramework, string projectPath, string solution
         {
             _logger.LogDebug("Started initial build using {0}", fullFramework ? "msbuild.exe" : "dotnet build");
 
-            projectPath = Path.GetDirectoryName(projectPath);
```

Comment:
The git history doesn't make it clear what was the point of this - basically if you have "Tests\Tests.csproj" it runs `dotnet build Tests` instead of `dotnet build Tests\Tests.csproj`. This used to work (rather by accident I'd say) but now to handle multiple project files in the folder we need to point to the specific file. IMO this is anyway clearer and removes an impurity from the code.